### PR TITLE
Updates: network reqs, comparing images, policies

### PR DESCRIPTION
**Pull request by ltagliaferri**

### CODE REVIEW

File: content/chainguard/administration/network-requirements.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+description: "Using Chainguard Images with firewalls, access control lists, and proxies"
 date: 2023-09-08T08:49:31+00:00
-lastmod: 2024-03-28T15:22:20+01:00
+lastmod: 2024-05-06T15:22:20+01:00
 draft: false
 aliases:
   - /chainguard/chainguard-images/reference/network-requirements/
   - /chainguard/chainguard-enforce/reference/network-requirements/
-tags: ["Chainguard Images", "Chainguard Enforce", "Product", "Reference"]
+tags: ["Chainguard Images", "Product", "Reference"]
```
Comment (SharpRake) [inline content/chainguard/administration/network-requirements.md:13]: nice! this is the last doc that still has the `Enforce` tag

File: content/open-source/sigstore/policy-controller/policies/using-policy-controller-to-verify-signed-chainguard-images.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
 ---
```
Comment (SharpRake) [inline content/open-source/sigstore/policy-controller/policies/using-policy-controller-to-verify-signed-chainguard-images.md:1]: Just looking at the lefthand nav, would this doc make more sense in the `Working with Images` bucket?

File: content/open-source/sigstore/policy-controller/how-to-install-policy-controller.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 ---
```
Comment (SharpRake) [inline content/open-source/sigstore/policy-controller/how-to-install-policy-controller.md:1]: not sure how this got missed in the audit but this doc has a section that points to Enforce docs which we ought to remove: https://deploy-preview-1558--ornate-narwhal-088216.netlify.app/open-source/sigstore/policy-controller/how-to-install-policy-controller/#options-for-continuous-verification

Comment (SharpRake) [reply content/open-source/sigstore/policy-controller/how-to-install-policy-controller.md:1]: Ah, I see now that you removed this same section from several guides below. Well here's another one that ought to get cut

**Review by SharpRake — APPROVED** [review]
LGTM! One doc still has the "Options for Continuous Verification" section and I left one food for thought comment on the organization, but otherwise these changes are :+1:

File: content/open-source/sigstore/policy-controller/policies/using-policy-controller-to-verify-signed-chainguard-images.md (hunk 2 above)
Comment (ltagliaferri) [reply content/open-source/sigstore/policy-controller/policies/using-policy-controller-to-verify-signed-chainguard-images.md:1]: I moved there

**Pull request merged**